packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx

```diff
@@ -23,10 +25,32 @@ export const getDocs: () => JSX.Element = () => {
       <Source code={ComplianceBannerText} />
       <Subheading>Use Compliance Banner</Subheading>
       <Source code={CallComponentText} />
-      <Title>Lobby Component</Title>
+      <Heading>Lobby Component</Heading>
       <Description>The Lobby component can be used for scenarios where the call is in a waiting state.</Description>
-      <Heading>Create a Lobby Component</Heading>
+      <Subheading>Create a Lobby Component</Subheading>
       <Source code={LobbyComponentText} />
+      <Heading>Inline Images</Heading>
+      <Subheading>Use "CallWithChat" Composite with Inline Image Support</Subheading>
+      <Description>
+        Currently, the UI library only supports inline images in a Teams Interop chat sent by the Teams user. To
+        lerverage this feature, you can use the "CallWithChat" composite.
+      </Description>
+      <Description>
+        You can try out the "CallWithChat" composite in the [CallWithChatComposite Basic
+        Example](./?path=/story/composites-call-with-chat-basicexample--basic-example).
+      </Description>
+      <Subheading>Use "MessageThread" component with Inline Image Support</Subheading>
+      <Description>
+        The following example shows how to implement "MessageThread" to render a message with protected images and where
+        you can provide custom authentication logic for the UI library to fetch and render images.
+      </Description>
+      <Description>
+        Noticing in the image tag, the ID attribute contains the same id as defined in the of "attachedFilesMetadata".
```

Comment:
might be better to change it to `<img>` tag, and probably can utilize "\`" quote for variable name like `id` etc... instead of just double quote "

Reply:
it doesn't like html element in this, tried to wrap it in `"` and `'` but getting the same error
![image](https://user-images.githubusercontent.com/109105353/231891113-6ef5be7d-3f38-4006-a940-ebb5769c1673.png)